src/kadmin/ktutil/ktutil.c

```diff
@@ -149,6 +150,7 @@ void ktutil_add_entry(argc, argv)
         }
         if ((strlen(argv[i]) == 2) && !strncmp(argv[i], "-k", 2)) {
             kvno = (krb5_kvno) atoi(argv[++i]);
+            use_kvno++;
```

Comment:
What's the use case for specifying kvno 0?  That's somewhat dangerous in our code base, as we use kvno 0 for "unspecified" in a lot of places, including the encoding of ASN.1 sequences that include kvno values.

Also, sneaking a second feature into a commit isn't great for our record-keeping, so I would probably want to separate this out.